lib_runtime/mirage_runtime.ml

```diff
@@ -119,6 +119,17 @@ let delay =
   in
   Arg.(value & opt int 0 doc)
 
+(** {3 Name} *)
+
+let name_k =
+  let doc =
+    Arg.info ~docs:Cmdliner.Manpage.s_common_options
+      ~doc:"Name of the unikernel" [ "name" ]
+  in
+  Arg.(value & opt string "no-name" doc)
+
+let name = Functoria_runtime.register_arg name_k
```

Comment:
What stands out to me is that `name` is the only runtime argument we register here in `Mirage_runtime`.

Reply:
exactly -- also the delay_key, ocaml_runtime etc. are used by the generated code, while the name is not. there's no need to put more stuff into the generated main.ml.